### PR TITLE
HONKS EXTERNALLY

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Emagging a honkbot is now even more horrifying.
#### Changelog

:cl:
rscadd: Emagging a honkbot will create unending suffering and terror.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
we don't have honkbots fully ported

**Comment by ghost** [comment]
honkbots are fully ported and exist, they're just not in the map

they are buildable

File: code/modules/mob/living/simple_animal/bot/honkbot.dm [hunk 1]
```diff
@@ -118,22 +118,43 @@
...
+	jitter = 20
+	flag = "energy"
+
+/obj/item/projectile/clownblast/syndicate/on_hit(atom/target, blocked = 0)
+	. = ..()
+	if(iscarbon(target))
+		var/mob/living/carbon/M = target
+		M.adjustEarDamage(0, 30)
+		M << "<font color='red' size='7'>HONK</font>"
+		M.SetSleeping(0)
```
Comment (Cruix) [inline code/modules/mob/living/simple_animal/bot/honkbot.dm:147]: Nice touch.

**Comment by Kmc2000** [comment]
Merge

NOW

**Comment by Alek2ander** [comment]
It should reuse banana peels instead of spawning an endless number.

**Comment by Super3222** [comment]
Make it slower when emagged to compensate for... the bananas.

**Comment by Cruix** [comment]
Fairly certain I will regret merging this at some point, but no one ever builds honkbots, let alone emags them.

**Pull request merged**